ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -4473,9 +4473,13 @@ private static bool NumberFormatsAreEqual(NumberingFormat nf, XLNumberFormatValu
         #region GenerateWorksheetPartContent
 
         private static void GenerateWorksheetPartContent(
-            WorksheetPart worksheetPart, XLWorksheet xlWorksheet, bool evaluateFormulae, SaveContext context)
+            WorksheetPart worksheetPart, XLWorksheet xlWorksheet, bool evaluateFormulae, bool consolidateConditionalFormatRanges, bool consolidateDataValidationRanges, SaveContext context)
         {
-            ((XLConditionalFormats)xlWorksheet.ConditionalFormats).Consolidate();
+            if(consolidateConditionalFormatRanges)
```

Comment:
Add a space after `if`

Reply:
To add to this, install the CodeMaid Visual Studio plugin and do a CodeMaid Clean on each file that you have altered. It will ensure the whitespace is in line with our convention.

Reply:
codemaid applied